domains/yanggui.json

```diff
@@ -0,0 +1,8 @@
+
+{
+    "description": "Describe the use of this subdomain",
+    "repo": "https://github.com/xuchao996",
+    "record": {
+        "URL": "https://yanggui.is-a-good.dev"
```

Comment:
cant blame more i cant access this site so..................................................